source/GameInterface/Services/BasicCultureObjects/BasicCultureObjectRegistry.cs

```diff
@@ -0,0 +1,39 @@
+﻿using GameInterface.Services.Registry;
+using System.Threading;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.Core;
+using TaleWorlds.ObjectSystem;
+
+namespace GameInterface.Services.BasicCultureObjects
+{
+    internal class BasicCultureObjectRegistry : RegistryBase<BasicCultureObject>
+    {
+        private const string IdPrefix = "CoopBasicCulture";
+        private static int InstanceCounter = 0;
+
+        public BasicCultureObjectRegistry(IRegistryCollection collection) : base(collection)
+        {
+        }
+
+        public override void RegisterAll()
+        {
+            var objectManager = MBObjectManager.Instance;
+
+            if (objectManager == null)
+            {
+                Logger.Error("Unable to register objects when CampaignObjectManager is null");
+                return;
+            }
+
+            foreach (var culture in objectManager.GetObjectTypeList<BasicCultureObject>())
+            {
+                RegisterExistingObject(culture.StringId, culture);
+            }
```

Comment:
Because ObjectManager uses MBObjectManager for MBObjectBase types, this should already be registered. I suspect that `RegisterExistingObject` is returning false when running the game rather than the tests.

Just make sure that they are not being registered twice